index.html

```diff
@@ -5,6 +5,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Studio Ghibli Companion App</title>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Titillium+Web:ital,wght@0,400;0,700;1,200&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
Any reason to add this? It doesn't seem like this font is used anywhere on this page.

Reply:
No real reason, I was thinking of using google fonts in the future, however, I probably won't have the time to use them.

I'll remove the link and do a git push.